Main.cs

```diff
@@ -107,25 +108,7 @@ private Result CreateResult(string keyword, SearchResult searchResult)
                         switch (searchResult.Type)
                         {
                             case ResultType.Folder:
-                                if (!_settings.LaunchHidden)
-                                {
-                                    Process.Start(_settings.ExplorerPath,
-                                        _settings.ExplorerArgs.Replace(Settings.DirectoryPathPlaceHolder, $"\"{path}\""));
-                                }
-                                else
-                                {
-                                    ProcessStartInfo startInfo = new ProcessStartInfo();
-                                    //Hide the process
-                                    startInfo.UseShellExecute = false;
-                                    startInfo.RedirectStandardOutput = true;
-                                    startInfo.WindowStyle = ProcessWindowStyle.Hidden;
-                                    startInfo.CreateNoWindow = true;
-                                    //Set file and args
-                                    startInfo.FileName = _settings.ExplorerPath;
-                                    startInfo.Arguments = _settings.ExplorerArgs.Replace(Settings.DirectoryPathPlaceHolder, $"\"{path}\"");
-                                    //Start the process
-                                    Process proc = Process.Start(startInfo);
-                                }
+                                _context.API.OpenDirectory(path);
```

Comment:
this has a setting to launch hidden, does the api code replicate the same behaviour?

Reply:
Actually I am not quite sure about launch hidden. Isn't that for launch everything hidden?

Reply:
There is an option in settings, i cant get my everything service to work in the sandbox. Can you check please.

![image](https://user-images.githubusercontent.com/26427004/147614818-b86b4aae-9e9f-4657-b36f-df2e63e1a335.png)

Reply:
weird. I didn't see that in my build.

Reply:
@taooceros is this one sorted?

Reply:
@taooceros is this one sorted?